packages/backend/prisma/seed/config/Categories.ts

```diff
@@ -0,0 +1,18 @@
+export const Categories: string[] = [
+  "Ноутбуки та комп'ютери",
```

Comment:
Use some constants and add categories to localization 
```
'LAPTOPS_AND_PC_CATEGORY_NAME',
```

In ua localization
```
LAPTOPS_AND_PC_CATEGORY_NAME: ''Ноутбуки та комп'ютери"
```

localization path packages/frontend/public/locales